diff --git a/src/big.ts b/src/big.ts
index 2222..3333 100644
--- a/src/big.ts
+++ b/src/big.ts
@@ -99,0 +100,20 @@
+// changed line 100
+// changed line 101
+// changed line 102
+// changed line 103
+// changed line 104
+// changed line 105
+// changed line 106
+// changed line 107
+// changed line 108
+// changed line 109
+// changed line 110
+// changed line 111
+// changed line 112
+// changed line 113
+// changed line 114
+// changed line 115
+// changed line 116
+// changed line 117
+// changed line 118
+// changed line 119
